### ISSUE TITLE
Retirement

### ISSUE THREAD

**Issue by AlgorithmX2** [ChiselsAndBits/Chisels-and-Bits#474]
Its been a long road, roughly 4 or so years since I came back after retiring from AE2 Dev, and made C&B, and FCB then later Ambidextrous.

If you know anything about my recent time constraints, then this might not come as a surprise, but likely you don't. I just don't have the time to work on these projects anymore. Over the past years I've slowly had less and less, combined with how I've drifted away from the modding community as a whole, I don't have the interest in these projects I used to.

I've been telling myself since 1.13 was announced that I would simply make the time, find it one way or another and just make it happen. However after doing research, looking into the new versions, I realize, that between my time constraints, and lack of interest, I no longer believe that I can find that time, or motivation.

I should have probably have done this a long time ago, as I've dreaded working on any modding project for some time now. The simple truth is, I just don't enjoy working on them anymore, and having not played Minecraft in roughly 3 years, just the idea of firing up java development tools and getting back into the code to fix or research something is just work to me now, worse it feels more arduous then my day job.

I had every intention of porting to 1.13, or 1.14, but at the end of the day I don't think I have it in me to finish that task. For being unable to keep my word on that matter, I apologize. But as I should have learned long ago, I push myself too hard, for far too long before realizing it was time to move on.

Its time that I hang up my mod developer cape for good this time.

Thanks to everyone who made this community a genuinely positive experience for me and others who have come and gone over the years.

TLDR; Sorry, but I'm retired, and overly dramatic.

---

If anyone is interested in continuing C&B, FCB, or Ambidextrous, I have no issues with you doing so, these mods have always been open source, and even though there hasn't been many people contributing, I still support my decision to keep them open and available if anyone is willing to put in the work to keep them going.

**Comment by Panduso** [comment]
I understand, if you don't want to you don't need to.

**Comment by alphapapa** [comment]
@AlgorithmX2 Thanks for your work on C&B.  It enabled my friend and I to make some unique creations that wouldn't have been possible otherwise.  Many hours of enjoyment and fond memories came from it.  I'll share a few here as a way of saying thanks: https://imgur.com/a/647gKHS

I don't blame you for wanting to get off the unbreak-the-mod-in-the-newest-Minecraft-version treadmill.  I wouldn't have the patience, especially if I weren't playing Minecraft anymore.  Best to face the situation and let someone else carry on.

And I hope that someone else will pick up the torch, and since it's free software, there's a good chance that will happen someday.  I think C&B is the most interesting, fundamental Minecraft mod there is, and with the incredible work you put into it, it works so well!  I never had any problems or crashes with it.  So it must be a good foundation to build on.

Best wishes in your future endeavors!

**Comment by Aeltumn** [comment]
Hiya!

I'd love to pick up the torch and continue Chisels & Bits and Flat Colored Blocks. I'm currently working a 1.14 version over at https://github.com/Aeltumn/Chisels-and-Bits-2, at the moment I've only managed to get the chiseled block rendering working but I'm hard at work at making a feature complete port/successor. (I've also got some ideas for extra features, like making FCB a part of C&B)

Thanks again for all your work on this mod, it's one of the best mods there is and I really hope I can do it justice in my adaptation.

**Comment by SonOfTheStars** [comment]
Id tip my hat to you Algo, if i had one. But still, thank you for the absolutely great work you did with CnB and FnB. Hope you really have a nice time going forward from this!

**Comment by Rosethorns** [comment]
All the best for the future Algo :) You have to do whats best for you. And thank you for the legacy you have left us with <3

**Comment by bogger33** [comment]
I wish you all the best, you've made multiple of the absolute best mods out there IMO, and always made them open source too, so the modding community has a lot to thank you for. Good luck in all your future endeavours!

**Comment by AnthonyPaulO** [comment]
Wow... I just discovered this mod recently and it's such an amazing piece of work, clearly a labor of love! I'm bummed to find out that the author is retiring, out of lack of motivation and interest in Minecraft no less!

@AlgorithmX2 Out of curiosity, what is it about Minecraft that you feel no longer does it for you? Is it lacking something? If so, would you share your thoughts? Or is it nothing particular to Minecraft, it's just that you are simply no longer interested in this type of game anymore? It would be very illuminating to say the least, especially from a veteran that had dedicated so much effort into the community. Thanks in advance!

**Comment by Michaelangel007** [comment]
@AlgorithmX2 Like Anthony I just recently discovered your cool Chisel and Bits mod and was blown away by how powerful it is!   Sorry to hear you are burnt out from it and sad to hear that you have no plans to continue it but I can understand wanting to take a break.  Mental health comes first.  Hope you have more time and energy for your future projects!

Is there anything you feel Mojang could have done in the last few years that would have a difference in helping to make you more excited to continue working on it?  i.e. Proper modding support?  Incorporating part of the C&B functionality into Minecraft? etc.

Thanks again for this cool mod.  It is a fantastic mod that has lots of neat ideas!

**Comment by gabistar** [comment]
I understand, you created great mods, but the game over time turns out to be sick (even for modders) good retirement :)
   I hope someone port the mods for recent versions ^^
